### PR TITLE
MTM-47207 Added widget sanitization info.

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

File: content/users-guide/cockpit-bundle/widgets-collection.md [hunk 1]
```diff
@@ -389,6 +389,11 @@ The "HTML" widget shows user-defined content. The content can be formatted using
 "Device" refers to the target device, as selected in the widget configuration parameter.<br>
 "fragment.property" refers to the properties of the respective device. To see the available property names, you can use the "Asset property" or "Asset table" widget and click **+Add property** in the widget configuration. This will show a table of supported properties. You can copy and paste the values from the column **Property**. Generated properties of these widgets are not available in the HTML widgets.
 
+The following sanitization options can be selected:
+ - strict - Does not allow any JS or angularjsj directives.
```
Comment (reey) [inline content/users-guide/cockpit-bundle/widgets-collection.md:393]: `angularjsj` typo

File: content/users-guide/cockpit-bundle/widgets-collection.md [hunk 2]
```diff
@@ -533,9 +538,10 @@ The "SCADA" widget provides a graphic representation of the status of a device.
 
 For details on the "SCADA" widget, refer to [Cloud Fieldbus > Monitoring status using the SCADA widget](/protocol-integration/cloud-fieldbus/#scada) in the *Protocol integration guide*.
 
-{{< c8y-admon-info >}}
-All SVG files are sanitized in order to remove malicious code.
-{{< /c8y-admon-info >}}
+The following sanitization options can be selected:
+ - strict - Does not allow any JS or angularjsj directives.
```
Comment (reey) [inline content/users-guide/cockpit-bundle/widgets-collection.md:542]: `angularjsj` typo

File: content/users-guide/cockpit-bundle/widgets-collection.md (hunk 1 above)
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/widgets-collection.md]: fixed

File: content/users-guide/cockpit-bundle/widgets-collection.md (hunk 2 above)
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/widgets-collection.md]: fixed

**Review by reey — APPROVED** [review]

File: content/users-guide/cockpit-bundle/widgets-collection.md [hunk 3]
```diff
@@ -389,6 +389,11 @@ The "HTML" widget shows user-defined content. The content can be formatted using
 "Device" refers to the target device, as selected in the widget configuration parameter.<br>
 "fragment.property" refers to the properties of the respective device. To see the available property names, you can use the "Asset property" or "Asset table" widget and click **+Add property** in the widget configuration. This will show a table of supported properties. You can copy and paste the values from the column **Property**. Generated properties of these widgets are not available in the HTML widgets.
 
+The following sanitization options can be selected:
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/widgets-collection.md:392]:
```suggestion
The following code sanitization options can be selected:
```

File: content/users-guide/cockpit-bundle/widgets-collection.md [hunk 4]
```diff
@@ -533,9 +538,10 @@ The "SCADA" widget provides a graphic representation of the status of a device.
 
 For details on the "SCADA" widget, refer to [Cloud Fieldbus > Monitoring status using the SCADA widget](/protocol-integration/cloud-fieldbus/#scada) in the *Protocol integration guide*.
 
-{{< c8y-admon-info >}}
-All SVG files are sanitized in order to remove malicious code.
-{{< /c8y-admon-info >}}
+The following sanitization options can be selected:
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/widgets-collection.md:541]:
```suggestion
The following code sanitization options can be selected:
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**